README.md

```diff
@@ -15,9 +15,20 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# Apache Celix [![Build Status](https://travis-ci.org/apache/celix.svg?branch=master)](https://travis-ci.org/apache/celix) [![Coverage Status](https://coveralls.io/repos/apache/celix/badge.svg?branch=master&service=github)](https://coveralls.io/github/apache/celix?branch=master) [![Coverity Scan Build Status](https://scan.coverity.com/projects/6685/badge.svg)](https://scan.coverity.com/projects/6685)
+# Status
+
+![Celix Ubuntu](https://github.com/apache/celix/workflows/Celix%20Ubuntu/badge.svg)
+
+![Celix MacOS](https://github.com/apache/celix/workflows/Celix%20MacOS/badge.svg)
+
+[![codecov](https://codecov.io/gh/apache/celix/branch/master/graph/badge.svg)](https://codecov.io/gh/apache/celix) 
+ 
+[![Coverity Scan Build Status](https://scan.coverity.com/projects/6685/badge.svg)](https://scan.coverity.com/projects/6685)
+
+# Apache Celix  
```

Comment:
Why have the badges in a separate section instead of:
Even with multiple badges this looks kinda nice. See as example https://github.com/apache/dubbo/blob/master/README.md
I agree to not have them after the `# Apache Celix` title.

```suggestion
# Apache Celix

![Celix Ubuntu](https://github.com/apache/celix/workflows/Celix%20Ubuntu/badge.svg)
![Celix MacOS](https://github.com/apache/celix/workflows/Celix%20MacOS/badge.svg)
[![codecov](https://codecov.io/gh/apache/celix/branch/master/graph/badge.svg)](https://codecov.io/gh/apache/celix) 
[![Coverity Scan Build Status](https://scan.coverity.com/projects/6685/badge.svg)](https://scan.coverity.com/projects/6685)
```